examples/seismic/acoustic/wavesolver.py

```diff
@@ -78,13 +87,22 @@ def forward(self, src=None, rec=None, u=None, m=None, save=None, **kwargs):
         Forward modelling function that creates the necessary
         data objects for running a forward modelling operator.
 
-        :param src: Symbol with time series data for the injected source term
-        :param rec: Symbol to store interpolated receiver data
-        :param u: (Optional) Symbol to store the computed wavefield
-        :param m: (Optional) Symbol for the time-constant square slowness
-        :param save: Option to store the entire (unrolled) wavefield
-
-        :returns: Receiver, wavefield and performance summary
+        Parameters
+        ----------
+        src :
```

Comment:
SparseTimeFunction, or maybe PointData (which is its subclass)? or...? @mloubout

Reply:
geometry, (like above in operator)

Reply:
Same as previous one yes geometry:.....

Reply:
what? there's no geometry here. Just src, rcv, ...

Reply:
Shouldn't the documentation go under **def __init__**